docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -102,7 +102,23 @@ Refer to the [Dagster Cloud Branch Deployments example](#example-2-dagster-cloud
 
 [Configurable Dagster objects](/concepts/configuration/config-schema) - such as ops, assets, resources, I/O managers, and so on - can accept configuration from environment variables.
 
-To access an environment variable as part of a Dagster configuration, use the following syntax:
+#### In Python code
+
+To access an environment variable as part of a Dagster configuration in Python code, use the following syntax:
+
+```python
+"PARAMETER_NAME": EnvVar("ENVIRONMENT_VARIABLE_NAME")
+```
+
+For example:
+
+```python
+"access_token": EnvVar("GITHUB_ACCESS_TOKEN")
+```
```

Comment:
should this be something like:
```
RunConfig(
    access_token=EnvVar("GITHUB_ACCESS_TOKEN")
)
```
this part is a bit confusing. maybe worth showing a full code snippet like passing config to a resource or op/asset